app/serializers/api/public/v1/serializable_location.rb

```diff
@@ -51,6 +51,10 @@ class SerializableLocation < JSONAPI::Serializable::Resource
         attribute :street_address_2 do
           @object.address2
         end
+
+        attribute :street_address_3 do
```

Comment:
If we're exposing this as a new field in the public api, we ought to add some notes in api docs as well.

Reply:
Done